bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -194,6 +197,16 @@ class App extends Component {
     window.ondragover = (e) => { e.preventDefault(); };
     window.ondrop = (e) => { e.preventDefault(); };
 
+    if (CONFIRMATION_ON_LEAVE) {
+      window.onbeforeunload = (event) => {
+        AudioService.muteMicrophone();
+        event.stopImmediatePropagation();
+        event.preventDefault();
+        // eslint-disable-next-line no-param-reassign
+        event.returnValue = '';
```

Comment:
It seems better to use this way instead of suppressing ESLint errors
```suggestion
        event.setAttribute('returnValue', '');
```